src/Behat/Gherkin/Node/ExampleNode.php

```diff
@@ -178,6 +187,25 @@ public function getOutlineTitle()
         return $this->outlineTitle;
     }
 
+    public function getName(): ?string
+    {
+        return "{$this->replaceTextTokens($this->outlineTitle)} #{$this->index}";
+    }
+
+    /**
+     * Returns the example row as a single string.
+     *
+     * You should normally not need this, since it is an implementation detail.
+     * If you need the individual example values, use {@see self::getTokens()}.
+     * To get the fully-normalised/expanded title, use {@see self::getScenarioTitle()}.
+     *
+     * @return string
+     */
+    public function getExampleText()
```

Comment:
I *think* we can strictly type this, since it's new & there is always text?

```suggestion
     */
    public function getExampleText(): string
```